erpnext/crm/doctype/lead/lead.json

```diff
@@ -140,6 +140,7 @@
    "oldfieldname": "status",
    "oldfieldtype": "Select",
    "options": "Lead\nOpen\nReplied\nOpportunity\nQuotation\nLost Quotation\nInterested\nConverted\nDo Not Contact",
+   "read_only": 1,
```

Comment:
Status updater only takes care of these status:

https://github.com/frappe/erpnext/blob/ecfa6344c02b18455637455742202c7fadf1b35d/erpnext/controllers/status_updater.py#L18-L23

What about some other status like Replied, Interested, Do no contact, etc?